ios/brave-ios/Sources/Brave/Extensions/AnimationView+Extensions.swift

```diff
@@ -0,0 +1,35 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Lottie
+import UIKit
+
+extension AnimationView {
+  static func showShredAnimation(on view: UIView, callback: @escaping () -> Void) {
```

Comment:
Is this code in a shared module because it's used in more than 1 place?

Reply:
Yes I use it in more than one place but suggestions are welcome how to share this code